tests/SpecTests/TransactionsSpecTest.php

```diff
@@ -36,6 +36,34 @@ class TransactionsSpecTest extends FunctionalTestCase
         'transactions/pin-mongos: unpin after transient error within a transaction and commit' => 'isMaster failpoints cannot be disabled',
     ];
 
+    private static array $incompleteLoadBalancerTests = [
+        'transactions/mongos-pin-auto: remain pinned after non-transient Interrupted error on insertOne' => 'libmongoc does not pin for load-balanced topology',
```

Comment:
This is alluded to in the description of [CDRIVER-4718](https://jira.mongodb.org/browse/CDRIVER-4718) but not actually documented anywhere.

The reason libmongoc does not pin is because a `mongoc_client_t` only has a single socket per endpoint (e.g. mongod, mongos, LB). Since clients themselves are pooled (unlike connections in other drivers), the idea of connecting pinning doesn't apply for a load balanced topology.